libse/TransportStream/DvbSubPes.cs

```diff
@@ -59,7 +59,7 @@ public DvbSubPes(byte[] buffer, int index)
 
             HeaderDataLength = buffer[index + 8];
 
-            if (buffer.Length < index + 9 + HeaderDataLength)
+            if (buffer.Length < index + 10 + HeaderDataLength)
```

Comment:
If we say 9 the buffer's length may be equal to 9 but in [line 67](https://github.com/SubtitleEdit/subtitleedit/pull/1934/files#diff-1eb1be2135f55969cc79857272e0c6a9R67) we are trying to access buffer from index 9.